### ISSUE TITLE
discussing issues

### ISSUE THREAD

**Issue by chgebhardt** [gklrajan/Zebranalysis#1]
i thought more something like this... :)

**Comment by gklrajan** [comment]
Okay! 👍

**Comment by gklrajan** [comment]
@chgebhardt : maybe you're notified now.

**Comment by chgebhardt** [comment]
yes i was, looks good to me. i made some changes to the file. can you see them? if yes then that is a problem  because i made them directly to the master... you would loose any control over your file...

**Comment by gklrajan** [comment]
eh, then the collaboration mode does not look good - I can see the changes, its directly on the master! (on a positive  note - all the previous versions will still be available in history).

And the notifications work now as I have commented on a new thread.

**Comment by chgebhardt** [comment]
or you trust me and i dont make any change on the master but a "swim bout analysis" branch...

but we should keep that in mind for later...  maybe do some research if this is normal github behavior or if we are just incapable for the moment lol
